backend-test/integration/user.test.js

```diff
@@ -0,0 +1,94 @@
+import 'babel-polyfill';
+import request from 'supertest';
+import faker from 'faker';
+import { expect } from 'chai';
+import mongoose from 'mongoose';
+import app from '../../server';
+import { Tokenizer } from '../../app/helpers/tokenizer';
+
+mongoose.Promise = global.Promise;
+
+const User = mongoose.model('User');
+let token;
+
+const mock = {
+  name: 'Hasstrup Ezekiel',
+  password: '12345',
+  username: 'hasstrupezekiel',
+  email: 'hasstrup@email.com'
+};
+
+describe('User endpoints', () => {
+  describe('Authentication', () => {
+    before(() => Promise.resolve(User.create(mock).then((user) => {
+      token = Tokenizer(user);
+    })
+      .catch((err) => {
+        throw err;
+      })));
+
+    it('POST /api/auth/login should return the user token along with the', (done) => {
+      try {
```

Comment:
Once again try catch is redundant, please remove it. I have mentioned this a lot before

Reply:
Will do. Thank you